### PR TITLE
Labs Solution for the Password generator project.

### PR DESCRIPTION
**Pull request by Quaye-David**
This pull-request contains files for the Week 2.2 labs on a dynamic password Generator app created using HTML Stack.

### CODE REVIEW

File: Labs Solutions/Week 2.1/style.css
Comment (Bilal-Abubakari) [inline Labs Solutions/Week 2.1/style.css:1]: I expected scss here

**Review by Bilal-Abubakari — COMMENTED** [review]
Your rubric is faulty

File: Labs Solutions/Week 2.1/modules/passwordGenerator.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    uppercase = true,
+    lowercase = true,
+    numbers = true,
+    symbols = false
+  } = {}) {
+    this.length = length;
+    this.options = { uppercase, lowercase, numbers, symbols };
+  }
+
+  generate() {
```
[the remark this answers is not in the document]
Comment (QuayeDNA) [reply Labs Solutions/Week 2.1/modules/passwordGenerator.js:20]: Fix the method and used an arrow function

File: Labs Solutions/Week 2.1/modules/strengthChecker.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    score = 2; // Default to Weak
+  }
+
+  // Additional criteria
+  if (options.repeat && /(\w)\1\1/.test(password)) {
+    score = Math.max(1, score - 1); // Penalize repeated characters
+  }
+
+  // Bonus points
+  if (password.length >= 14 && charTypesCount === 4) {
```
[the remark this answers is not in the document]
Comment (QuayeDNA) [reply Labs Solutions/Week 2.1/modules/strengthChecker.js:47]: Fixed the faulty rubric and re-arranged the conditional logics

**Pull request merged**